pages/profiles/cramartef.md

```diff
@@ -0,0 +1,14 @@
+# Fetra Marc Humbert Rahajason  
+
+Madagascar/GMT+3.00, East Africa Time.  
+
+I am an OLE leader and Education consultant.
+---
+**Language skills**
+* Speak both french and English.
```

Comment:
@cramartef the layout seems a little bit wired on [rawgit](https://rawgit.com/cramartef/cramartef.github.io/Branch0-cramartef/#!pages/profiles/cramartef.md), try to add a line break between sub-heading and bucket list. Other than that, it looks good to me